apps/api/src/info-request/info-request.controller.ts

```diff
@@ -0,0 +1,40 @@
+import { Controller, Get, Post, Body, Patch, Param, Delete } from '@nestjs/common';
+import { InfoRequestService } from './info-request.service';
+import { CreateInfoRequestDto } from './dto/create-info-request.dto';
+import { UpdateInfoRequestDto } from './dto/update-info-request.dto';
+import { Public } from 'nest-keycloak-connect';
+
+@Controller('info-request')
+export class InfoRequestController {
+  constructor(private readonly infoRequestService: InfoRequestService) {}
+
+  @Post()
+  @Public()
+  create(@Body() createInfoRequestDto: CreateInfoRequestDto) {
+    return this.infoRequestService.create(createInfoRequestDto);
+  }
+
+  @Get('list')
+  @Public()
+  findAll() {
+    return this.infoRequestService.findAll();
+  }
+
+  @Get(':id')
+  @Public()
+  findOne(@Param('id') id: string) {
+    return this.infoRequestService.findOne(id);
+  }
+
+  @Patch(':id')
+  @Public()
```

Comment:
Let's make all editorial endpoints actions private, so noone is able to edit a info request without permissions

```suggestion
```